cmd/clusterctl/cmd/rollout/undo.go

```diff
@@ -66,6 +67,7 @@ func NewCmdRolloutUndo(cfgFile string) *cobra.Command {
 		"Context to be used within the kubeconfig file. If empty, current context will be used.")
 	cmd.Flags().StringVarP(&undoOpt.namespace, "namespace", "n", "", "Namespace where the resource(s) reside. If unspecified, the defult namespace will be used.")
 	cmd.Flags().Int64Var(&undoOpt.toRevision, "to-revision", undoOpt.toRevision, "The revision to rollback to. Default to 0 (last revision).")
+	cmd.Flags().BoolVarP(&undoOpt.force, "force", "f", undoOpt.force, "Force rollback without validating the Kubernetes version skew policy.")
```

Comment:
Just to be clear, the rename was only for the func parameter, I like the `--force` flag as it is.